### PR TITLE
further refinement of sig-* names for test ownership

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: test/extended/controller_manager/deploy_trigger.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[sig-devex][Feature:OpenShiftControllerManager]", func() {
+var _ = g.Describe("[sig-workloads][Feature:OpenShiftControllerManager]", func() {
```
Comment (smarterclayton) [inline test/extended/controller_manager/deploy_trigger.go]: `[sig-apps]` is their upstream

File: test/extended/image_ecosystem/s2i_php.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
-var _ = g.Describe("[sig-devex][Feature:ImageEcosystem][php][Slow] hot deploy for openshift php image", func() {
+var _ = g.Describe("[sig-examples][Feature:ImageEcosystem][php][Slow] hot deploy for openshift php image", func() {
```
Comment (smarterclayton) [inline test/extended/image_ecosystem/s2i_php.go]: there is no such thing as `sig-examples`, please keep this devex.  I don't think "examples" would ever be a team we have.

**Comment by smarterclayton** [comment]
/hold

have to fix verify

**Comment by bparees** [comment]
bindata fixed, reasserting lgtm

/hold cancel

**Pull request merged**